projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss

```diff
@@ -1262,6 +1262,7 @@
 
     %form-group-textarea--disabled {
         color: var-get($theme, 'disabled-text-color') !important;
+        cursor: default;
```

Comment:
Hm, I wonder if instead of being specific about the cursor type when the form-group is disabled, can't we just do this, insdead:

```scss
%form-group-bundle--hover:not(%form-group-bundle--disabled) {
    cursor: pointer;
    ...
}
```